schema/deploy/warehouse/target/data.sql

```diff
@@ -121,10 +121,13 @@ with target_lineage (identifier, lineage) as (
         ('S. pneumoniae_APZTD4A',       'Streptococcus_pneumoniae'),
         ('S.pneumoniae',                'Streptococcus_pneumoniae'),
 
-        -- Targets reported by Ellume & Cepheid using SNOMED CT
-        ('http://snomed.info/id/181000124108', 'Influenza.A'),
-        ('http://snomed.info/id/441345003',    'Influenza.B'),
-        ('http://snomed.info/id/441278007',    'RSV')
+        -- Snomed CT targets used by Ellume, Cepheid, UW Retros
+        ('http://snomed.info/id/440930009',        'Adenovirus'),
+        ('http://snomed.info/id/1240581000000104', 'Human_coronavirus.2019'),
```

Comment:
Ugghh I forgot this SNOMED id is specific to the UK SNOMED system, not the international one. there's a comment about it in the `redcap-det uw-retrospectives` command, but I forgot it ended up creating a full URL given the SNOMED id... Do you think this is fine to leave for now, or should the `redcap-det` command be updated to use a different URL?

Reply:
It should be updated, but let's leave it for now unless you don't think it'll take long. None of our other refs make the SNOMED edition or version explicit either.